README.md

```diff
@@ -0,0 +1,381 @@
+
+
+* Does my tariff depend not only on the duration of the trip, but also on the time of day (e.g. you pay more if you
+  travel between eight and nine than if you travel between ten and eleven)?
+
+  -> Then your tariff is a [TimeBasedTariff](#timebasedtariff).
+
+
+
+
+All tariffs are based on a modular slot system. The difference between the three tariff models is only the definition of
+the slots and the calling of the slots. Each slot must be assigned a rate. The rate is used for the actual calculation
+of the price. There are also different types of rates:
+
+* The [FixedRate](#fixedrate) is the simpler form, with which one can specify a fixed amount, which is computed, as soon
+  as this Slot is called.
+
+
+* With the [TimeBasedRate](#timebasedrate) you can define an interval in which a fixed amount is calculated. With each
+  repetition of this interval a fixed amount is added to it. The interval does not have to be completely finished, the
+  amount is added when entering the interval.
+
+### SlotBasedTariff
+
+A SlotBasedTariff is structured in such a way that the duration of the trip is split into slots. The slots must start at
+zero and continue seamlessly, because all slots are used until the end of the trip to calculate the final price. The
+price is calculated for each slot individually and then added up.
+
+###### Example 1
+
+```JSON
+{
+  "type": "SlotBasedTariff",
+  "id": 1,
+  "currency": "EUR",
+  "rates": [
+    {
+      "type": "FixedRate",
+      "id": 2,
+      "currency": "EUR",
+      "price": {
+        "credit": 100
+      }
+    },
+    {
+      "type": "TimeBasedRate",
+      "id": 3,
+      "currency": "EUR",
+      "interval": {
+        "timeAmount": 90,
+        "timeUnit": "Minutes"
+      },
+      "pricePerInterval": {
+        "credit": 100
+      }
+    }
+  ],
+  "slots": [
+    {
+      "rate": 2,
+      "start": {
+        "timeAmount": 0,
+        "timeUnit": "MINUTES"
+      },
+      "end": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    },
+    {
+      "rate": 3,
+      "start": {
+        "timeAmount": 2,
+        "timeUnit": "HOURS"
+      }
+    }
+  ]
+}
+```
+
+This tariff has two slots with an associated rate. The first slot begins at the start of the trip and covers the first
+two hours. It does not matter if the bike is rented for ten minutes or two hours. During this time, one euro is charged.
+The second slot starts at two hours and is not limited. The TimeBasedRate belongs to this slot. If you now ride for
+three hours, one euro from the second slot is added to the one euro from the first slot, because the interval was
+started once. If you travel for five hours, three euros from the second slot are added to the one euro from the first
+slot, because the interval was started three times.
+
+It is also possible to define a **billing interval**. This causes that after this interval the slots are calculated from
```

Comment:
How about something like:
> It is also possible to define **billing-interval**. This will divide/window the time of the calculation-interval and repeat applying the tariff calculation on each time window.

Reply:
I think the billing interval is a big feature and in the current description, the weight of it comes a little short. What do you think?